readthedocs-theme/templates/readthedocs/homepage.html

```diff
@@ -3,14 +3,119 @@
 {% block body_id %}home{% endblock %}
 
 {% block content %}
-  <div class="ui container">
-    <div class="ui segment">
-      <h1 class="ui header">
-        {{ page.title }}
-        {% if page.subtitle %}<div class="sub header">{{ page.subtitle }}</div>{% endif %}
-      </h1>
-
-      {{ page.content }}
+
+  <!-- Callout -->
+  <section class="ui vertical segment padded" id="callout">
+
+    <div class="ui container">
+      <div class="ui grid centered middle aligned">
+
+        <div class="row">
+
+          <div class="column seven wide computer nine wide tablet sixteen wide mobile">
+            <h1 class="ui header massive">
+              <span class="ui header">Continuous documentation for technical projects</span>
+            </h1>
+            <div class="ui basic vertical padded huge segment">
+              <p>
+                Make documentation part of your project workflow with simplified processes that automate building, versioning and hosting of your docs.
+              </p>
+            </div>
+            
+            <div>
+              <a href="https://docs.readthedocs.io/page/intro/getting-started-with-sphinx.html" class="ui button large secondary gradient fade">Start your docs</a>
+              <a href="./product.html" class="ui button large secondary basic">Learn more</a>
+            </div>
+          
+          </div>
+
+          <div class="column seven wide computer nine wide tablet sixteen wide mobile">
+            <img src="/theme/img/shutterstock-concept-of-user-manual-1315141181-2.jpg"/>
+          </div>
+
+        </div>
+
+      </div>
     </div>
-  </div>
+
+  </section>
+  
+  <!-- Announcements -->
+  {% include 'readthedocs/includes/announcements.html' %}
+  
+  <!-- Infographics -->
+  <section class="ui vertical segment light padded" id="infographics">
+
+    <div class="ui container">
+
+      <div class="ui grid centered">
+        <div class="row">
+          <div class="column fourteen wide computer sixteen wide tablet">
+        
+            <h2 class="ui header">How it works</h2>
+
+            <div class="ui grid stackable three column">
+
+              <div class="column center aligned">
+                <div class="ui image">
+                  <img src="/theme/img/infographic-01.png"/> 
+                </div>
+                <p>
+                  Write your documentation easily, using .md or .rst, directly in your own IDE, as you work on what maters: your project.
```

Comment:
Hrm. I noted in the previous PR that this should be Markdown and reStructuredText, not `.md or .rst`. Seems we missed some of the review feedback in the previous PR. Is this not yet ready for review?